index.js

```diff
@@ -6,8 +6,10 @@ const rootURI = 'https://www.googleapis.com';
 export const refreshTokenURI = 'https://www.googleapis.com/oauth2/v4/token';
 const uploadExistingURI = id =>
     `${rootURI}/upload/chromewebstore/v1.1/items/${id}`;
-const publishURI = (id, target) =>
-    `${rootURI}/chromewebstore/v1.1/items/${id}/publish?publishTarget=${target}`;
+const publishURI = (id, target, deployPercentage) => (
```

Comment:
Functions shouldn't accept more than 1 optional parameter, can you turn this into an object? `{id, target, deployPercentage}`

Reply:
Yes, sure.